src/hera/workflows/runner.py

```diff
@@ -76,6 +84,17 @@ def _is_str_kwarg_of(key: str, f: Callable):
         return False
 
 
+def map_kwargs(function: Callable, kwargs: dict):
```

Comment:
Tiny suggestion because we're changing the kwarg keys only. Function is 💯 
```suggestion
def map_kwargs_keys(function: Callable, kwargs: dict):
```